resources/views/auth/email/forgot.blade.php

```diff
@@ -55,7 +55,7 @@
                                         type="text"
                                         class="form-control form-control-lg bg-glass text-white"
                                         name="username"
-                                        maxlength="15"
+                                        maxlength="30"
```

Comment:
There's a config value that specifies max name length: https://github.com/pixelfed/pixelfed/blob/c68fb2e771d0e1bc5bb1e9eaaaa1e17fb5d3b71f/config/pixelfed.php#L116-L120.

Can we re-use it here to avoid duplication?